offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/update/UpdatePersonView.groovy

```diff
@@ -32,6 +38,69 @@ class UpdatePersonView extends CreatePersonView{
     private void adjustViewElements() {
         submitButton.caption = "Update Person"
         abortButton.caption = "Abort Person Update"
+
+        //add a grid
+        affiliations = new Grid<>()
+        generateAffiliationGrid()
+        this.addComponent(affiliations,2)
+        //add the add button
```

Comment:
Another Idea would be to make the content in the grid non selectable since it's just an overview. 
```suggestion
affiliations.setSelectionMode(Grid.SelectionMode.NONE)
        //add the add button
```
Alternatively one could think about linking the selection to the organisation and address addition selection fields.